Minesweeper.cpp

```
@@ -4,114 +4,315 @@
+	};
+	
+	class MinesweeperGrid : public GameBoard::Grid<Tile> 
+	{
+	
+		typedef std::vector<std::vector<Tile>> TileGrid;
+		bool m_isBombTriggered = false;
 
+	private:
 
-		if (point.x - 1 >= 0 && board[point.y][point.x - 1].tileContent == Contents::BOMB) count++; // Middle Left
+		int GetNumberOfBombsAroundPoint(TileGrid board, IntVector2 point) const
```

cppcheck (2.10-2)

[performance] Function parameter 'board' should be passed by const reference.

(passedByValue)